Community-history (#8)
blitz-js/fr.blitzjs.com: app/pages/docs/community-history.mdx

---
title: Histoire de la Communauté
sidebar_label: Historique
---

Tout au long de l'année 2019, Brandon Bayer souhaitait quelque chose qui ressemblait à Ruby on
Rails mais adapté à Javascript et React. Si vous cherchez l'historique Twitter, vous tomberez
sur lui tweetant à ce propos assez souvent, disant des choses comme "_que quelqu'un
le fasse pour que je n'ai pas à le faire._" 😂

> Finalement en Février 2020, Brandon a eu un coup de téléphone d'un ami développeur de chez Ruby on Rails.

Reviewer: ```suggestion
Finalement en Février 2020, Brandon a reçu un coup de téléphone d'un ami développeur Ruby on Rails.
```

Cet appel lui a donné la motivation d'enfin essayer de faire Rails pour

> React. Cela semblait être le bon moment avec toutes les nouvelles features de Next.js et
> Prisma 2 qui venait d'être publié en beta.

Reviewer: ```suggestion
React. Cela semblait être le bon moment avec toutes les nouvelles fonctionnalités de Next.js ainsi
que Prisma 2 qui venait d'être publié en beta.
```

Une semaine plus tard, Brandon
[annonce officiellement le début de Blitz le 17 Février 2020](https://twitter.com/flybayer/status/1229425878481793024).

> À ce temps, il n'avait que quelques centaines de lignes de code prototype. Mais l'idée

Reviewer 2: ```suggestion
À ce moment, il n'avait que quelques centaines de lignes de code prototype. Mais l'idée
```

> a raisonné auprès de tant de gens, que beaucoup ont sauté le pas et sont venus aider

Reviewer 2: ```suggestion
a résonné auprès de tant de gens, que beaucoup ont sauté le pas et sont venus aider
```
